eng/pipelines/coreclr/templates/crossgen-comparison-job.yml

```diff
@@ -78,6 +89,16 @@ jobs:
         artifactName: '$(buildProductArtifactName)'
         displayName: 'product build'
 
+    # Optionally download live-built libraries
+    - ${{ if ne(parameters.liveLibrariesBuildConfig, '') }}:
+      - template: /eng/pipelines/common/download-artifact-step.yml
+        parameters:
+          unpackFolder: $(librariesDownloadDir)
+          cleanUnpackFolder: false
+          artifactFileName: '$(librariesBuildArtifactName)$(archiveExtension)'
+          artifactName: '$(librariesBuildArtifactName)'
+          displayName: 'live-built libraries'
+
```

Comment:
I think we'll also need $(librariesOverrideArg) below at line 116 (build-test script invocation) unless I'm mistaken.

Reply:
Hmm, perhaps not, looks like you've already changed this logic.